source/content/guides/decoupled/overview/01-introduction.md

```diff
@@ -40,7 +38,7 @@ Decoupled Architecture is a site architecture that combines the speed and agilit
 
 Pantheon Front-End Sites provide users with tools that improve the experience of building a decoupled frontend that sources data from a CMS backend.
 
-With decoupled sites, a CMS site can be linked to a single decoupled site, multiple sites, or the user does not need a CMS at all and can link directly to a frontend application.
+With decoupled sites, a CMS site can be linked to a single decoupled site, multiple sites, or you can not use a CMS at all and link directly to a frontend application.
```

Comment:
when used as a noun, it's `back end` or `front end`. But it's almost always clearer to add a more specific noun. So instead of `Drupal is the back end` it's clearer to write `Drupal is the back-end CMS`. and `Drupal is the backend` is just incorrect.

Reply:
@stevector I think the tweet is referencing the Microsoft Style guide. Backend an frontend are also considered acceptable, and allows us to avoid the hyphen to create greater visual distinction between the product name and the use of the general technical term (Front-End Sites and frontend site).